### PR TITLE
fine-toothed comb pass

### PR DESCRIPTION
**Pull request by petrelharp**

### CODE REVIEW

File: ms/spaceness.tex [hunk 1]
```diff
@@ -176,14 +178,14 @@ \subsection{Modeling Evolution in Continuous Space}
 %%%%%%%%%%%%
 \subsection{A Forward-Time Model of Evolution in Continuous Space}
 
-We simulated populations using the non-Wright-Fisher module in the program SLiM v3.1 \citep{Haller2019}. 
+We simulated populations using the program SLiM v3.1 \citep{Haller2019}.
```
Comment (petrelharp) [inline ms/spaceness.tex]: nonWF goes without saying

File: ms/spaceness.tex [hunk 2]
```diff
@@ -176,14 +178,14 @@ \subsection{Modeling Evolution in Continuous Space}
...
-We simulated populations using the non-Wright-Fisher module in the program SLiM v3.1 \citep{Haller2019}. 
+We simulated populations using the program SLiM v3.1 \citep{Haller2019}. 
 Each time step consists of three stages: reproduction, dispersal, and mortality. 
 To reduce the parameter space we use the same parameter, denoted $\sigma$, 
 to modulate the spatial scale of interactions at all three stages by adjusting the standard deviation of the corresponding Gaussian functions. 
 As in previous work \citep{Wright1943,Ringbauer2017}, $\sigma$ is equal to the mean parent-offspring distance.  
 
 At the beginning of the simulation individuals are distributed uniformly at random on a continuous, square landscape. 
-Individuals are hermaphroditic, and each time step, each produces a Poisson number of offspring with mean $1/L$ where $L$ is the expected lifespan.
+Individuals are hermaphroditic, and each time step, each produces a Poisson number of offspring with mean $1/L$.
```
Comment (petrelharp) [inline ms/spaceness.tex]: It is confusing to say "expected lifetime" here.

File: ms/spaceness.tex [hunk 3]
```diff
@@ -272,9 +274,11 @@ \subsection{Demographic Parameters}
 To estimate generation times, we stored ages of the parents of every new individual born across 200 timesteps, 
 after a 100 generation burn-in, and took the mean. 
 To estimate variance in offspring number, 
-we tracked the number of offspring for all individuals for 100 timesteps following a 100-timestep burn-in period, subset the resulting table to include only the last timestep recorded for each individual, and calculated the variance in number of offspring across all individuals in timesteps 50-100.
+we tracked the lifetime total number of offspring for all individuals for 100 timesteps following a 100-timestep burn-in period,
+% subset the resulting table to include only the last timestep recorded for each individual,
+and calculated the variance in number of offspring across all individuals in timesteps 50-100.
```
Comment (petrelharp) [inline ms/spaceness.tex]: we just need to say what we calculate, not precisely how

File: ms/spaceness.tex [hunk 4]
```diff
@@ -292,17 +296,16 @@ \subsection{Sampling}
 \subsection{Summary Statistics}
 We calculated the site frequency spectrum and a set of 18 summary statistics (Table \ref{table:sumstats}) from 60 diploid individuals sampled from the final generation of each simulation using the python package scikit-allel \citep{Miles2017}. 
 Statistics included common single-population summaries including mean pairwise divergence ($\pi$), inbreeding coefficient ($F_{IS}$), and Tajima's $D$, 
-as well as an isolation-by-distance regression of genetic distance ($D_{xy}$) against the logarithm of geographic distance analogous to \cite{Rousset1997}'s approach, 
-which we summarized as the correlation coefficient between the logarithm of the spatial distance and the proportion of identical base pairs across pairs of individuals. 
+as well (motivated by \citet{Rousset1997}'s results)
+the correlation coefficient between the logarithm of the spatial distance and the proportion of identical base pairs across pairs of individuals. 
+% as an isolation-by-distance regression of genetic distance ($D_{xy}$) against the logarithm of geographic distance analogous to \citet{Rousset1997}'s approach, which we summarized as
```
Comment (petrelharp) [inline ms/spaceness.tex]: said the same thing twice

File: ms/spaceness.tex [hunk 5]
```diff
@@ -292,17 +296,16 @@ \subsection{Sampling}
...
-as well as an isolation-by-distance regression of genetic distance ($D_{xy}$) against the logarithm of geographic distance analogous to \cite{Rousset1997}'s approach, 
-which we summarized as the correlation coefficient between the logarithm of the spatial distance and the proportion of identical base pairs across pairs of individuals. 
+as well (motivated by \citet{Rousset1997}'s results)
+the correlation coefficient between the logarithm of the spatial distance and the proportion of identical base pairs across pairs of individuals. 
+% as an isolation-by-distance regression of genetic distance ($D_{xy}$) against the logarithm of geographic distance analogous to \citet{Rousset1997}'s approach, which we summarized as 
 
-Following recent studies that showed strong signals for dispersal and demography in the distribution of shared haplotype block lengths \citep{Ringbauer2017,Baharian2016}, we also calculated various summaries of the distribution of pairwise identical-by-state (IBS) block lengths among sampled chromosomes,
-defined to be the set of distances between adjacent sites that differ between the two chromosomes
-of each diploid. \revpoint{1}{1}
```
Comment (petrelharp) [inline ms/spaceness.tex]: I wrote this wrongly the first time.

File: ms/spaceness.tex [hunk 6]
```diff
@@ -328,16 +331,17 @@ \subsection{Association Studies}
...
 
 In all simulations, the phenotype of each individual
-is determined by adding independent Gaussian noise with mean zero \revpoint{1}{5}
-to a mean that may depend on spatial position such that the mean of spatially-correlated phenotypes varies by two standard deviations across the landscape.\llabel{phenotypescalingsd}
-We then adjust the geographic pattern of mean phenotype to create four spatially autocorrelated environmental influences on phenotype.
+is determined by drawing from a Gaussian distribution with standard deviation 10 \revpoint{1}{5}
+and a mean that may depend on spatial position.
+In spatially varying models, the mean phenotype differs by two standard deviations across the landscape.
+We then adjust the geographic pattern of mean phenotype to create four types of spatially autocorrelated environmental influences on phenotype.
```
Comment (petrelharp) [inline ms/spaceness.tex]: I think this is more clear now?

File: ms/spaceness.tex [hunk 7]
```diff
@@ -389,10 +395,6 @@ \subsection{Demographic Parameters and Run Times}
 for neighborhood sizes larger than 100.
 Census sizes range from around 14,000 at low $\sigma$ in the random mating model to 10,000 for both models when neighborhood sizes approach 1,000.
 The scaling of census sizes in both random-mating and spatial models appears to be related to two consequences of the spatial competition function: the decline of fitness at range edges, which effectively reduces the habitable area by one $\sigma$ around the edge of the map and so results in a smaller habitable area at high $\sigma$ values; and variation in the equilibrium population density given varying competition radii.
-Furthermore, lack of available mates contributes to lower census sizes at small $N_W$
-for the spatial model
-(We calculate $N_W$ for both in the same way,
-even though the number of ``available mates'' is much larger with random mating). \llabel{randomNW}
```
Comment (petrelharp) [inline ms/spaceness.tex]: you already said this below; I didn't need to add it

File: ms/spaceness.tex [hunk 8]
```diff
@@ -418,9 +420,9 @@ \subsection{Demographic Parameters and Run Times}
...
+Run times for our model scale approximately linearly with neighborhood size (Figure \ref{fig:runtimes}), with the lowest neighborhood sizes reaching 30N generations in around a day and those with neighborhood size approaching 1,000 requiring up to three weeks of computation. As currently implemented running simulations at neighborhood sizes more than 1,000 to coalescence is likely impractical, though running these models for more limited timescales and then ``recapitating'' the simulation using reverse-time simulation from the resulting tree sequence in msprime is possible \citep{haller2019treesequence}.  \llabel{runtimesxn}
```
Comment (petrelharp) [inline ms/spaceness.tex]: calling it "recapitating" - maybe jargon? - and citing for it

File: ms/spaceness.tex [hunk 9]
```diff
@@ -440,17 +442,24 @@ \subsection{Impacts of Continuous Space on Population Genetic Summary Statistics
...
 \includegraphics[width=.5\textwidth]{figures/ibs_cumdists_w_rm.pdf}
-\caption{Cumulative distributions for IBS tract lengths per pair of individuals at different geographic distances, across three neighborhood sizes (NS).}
+\caption{
+    Cumulative distributions for IBS tract lengths per pair of individuals at different geographic distances, across three neighborhood sizes (NS).
+    Nearby pairs (red curves) share many more long IBS tracts than do distant pairs (blue curves),
+    except in the random mating model.
+    The distribution of long IBS tracts between nearby individuals are very similar across neighborhood sizes,
+    but distant individuals are much more likely to share long IBS tracts at high neighborhood size
+    than at low neighborhood size.
+}
```
Comment (petrelharp) [inline ms/spaceness.tex]: I put in some more description of how to interpret the figure.

File: ms/spaceness.tex [hunk 10]
```diff
@@ -462,30 +471,40 @@ \subsubsection{IBS tracts and correlations with geographic distance}
...
 \begin{figure*}[p]
 \centering
 \includegraphics[width=\textwidth]{figures/distance_by_DAF.pdf}
-\caption{Trends in the distance among allele copies at varying derived allele frequencies and neighborhood sizes (NS).}
+\caption{
+    Spatial spread of rare alleles by neighborhood size (NS):
+    Each plot shows the distribution (across derived alleles and simulations)
+    of average pairwise distance between individuals carrying a focal derived allele
+    and derived allele frequency.
+}
```
Comment (petrelharp) [inline ms/spaceness.tex]: also, here.

**Comment by petrelharp** [comment]
A bunch of latex cleanup and a bit of rewording...

File: ms/spaceness.tex (hunk 6 above)
Comment (andrewkern) [reply ms/spaceness.tex]: can we just say "...a mean that depends..." .. the "may" makes this a bit more confusing than it needs to be

**Pull request merged**